components/Ribbon/samples/RibbonCustomSample.xaml

```diff
@@ -0,0 +1,245 @@
+<!--  Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information.  -->
+<Page x:Class="RibbonExperiment.Samples.RibbonCustomSample"
+      xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+      xmlns:controls="using:CommunityToolkit.WinUI.Controls"
+      xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+      xmlns:local="using:RibbonExperiment.Samples"
+      xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+      mc:Ignorable="d">
+
+    <!--
+        TODO: The generic.xaml file from the code project is not imported when the sample is loaded in the gallery app.
+        As a mitigation, we are manually importing the style dictionary here.
+    -->
+    <Page.Resources>
+        <ResourceDictionary Source="ms-appx:///CommunityToolkit.WinUI.Controls.Ribbon/RibbonStyle.xaml" />
+    </Page.Resources>
+
+    <StackPanel Spacing="16">
+        <StackPanel.Resources>
+            <Style BasedOn="{StaticResource DefaultAppBarButtonStyle}"
+                   TargetType="AppBarButton">
+                <Setter Property="LabelPosition" Value="Collapsed" />
+                <Setter Property="Width" Value="48" />
+                <Setter Property="Margin" Value="8" />
+                <Setter Property="Height" Value="48" />
+            </Style>
+        </StackPanel.Resources>
+
+        <controls:Ribbon HorizontalAlignment="Stretch">
+
+            <controls:RibbonCollapsibleGroup AccessKey="AB"
+                                             CollapsedAccessKey="AA"
+                                             Label="Edit"
+                                             Style="{StaticResource RibbonLeftCollapsibleGroupStyle}">
+                <controls:RibbonCollapsibleGroup.IconSource>
+                    <SymbolIconSource Symbol="Add" />
+                </controls:RibbonCollapsibleGroup.IconSource>
+
+                <Grid>
+                    <Grid.ColumnDefinitions>
+                        <ColumnDefinition Width="Auto" />
+                        <ColumnDefinition Width="Auto" />
+                        <ColumnDefinition Width="Auto" />
+                    </Grid.ColumnDefinitions>
+                    <Grid.RowDefinitions>
+                        <RowDefinition Height="*" />
+                        <RowDefinition Height="*" />
+                    </Grid.RowDefinitions>

```

Comment:
Indeed, it could be interesting to have a dedicated panel here to avoid repeating the `Grid` and all its setup. 
We can probably add a `MaxLinesOrColumns` property to the `WrapPanel`.